src/java.base/share/classes/jdk/internal/loader/RawNativeLibraries.java

```diff
@@ -35,8 +35,7 @@
 import java.util.Objects;
 import java.util.concurrent.ConcurrentHashMap;
 
-import static jdk.internal.loader.NativeLibraries.*;
-
+import static jdk.internal.loader.NativeLibraries.findEntry0;
```

Comment:
We have `NativeLibraries` and `RawNativeLibraries` which are unrelated classes. Then we have `NativeLibraryImpl` and `RawNativeLibraryImpl` which shared a common super type (`NativeLibrary`). I find it mildly asymmetric to put the `find0` native method in `NativeLibraries` since that's not the supertype of `RawNativeLibraries`. Is there common code between the two XYZLibraries classes that can be shared? If so, maybe `find0` can go in that common supertype - if not, maybe it's better to put `find0` in `NativeLibrary` itself?

Reply:
I agree it's asymmetric to put the native method in `NativeLibraries`.   `NativeLibrary` would be a better place for `findEntry0`.  I will also turn `NativeLibrary` to an abstract class to make `findEntry0` is package-private static method.